tendrl/commons/objects/definition/master.yaml

```diff
@@ -820,6 +820,7 @@ namespace.tendrl:
           - afr_quorum_state
           - afr_subvol_state
           - georep_status
+          - cluster_status
```

Comment:
why have you added this?

Reply:
https://github.com/Tendrl/node-agent/pull/742/files#diff-3dfeadf899a1fe4e98d3f57feef0b4daR84

@shtripat is added new alert called cluster_status for raising new peer node identify logic,
alert messages should comes under at least any one classification. otherwise alert count won't be increased